Gemfile

```diff
@@ -9,7 +9,9 @@ end
 gem "puma"
 gem "rails", "~> 5.1.5"
 
+gem "feedjira"
 gem "font-awesome-sass"
+gem "httparty"
```

Comment:
have we considered using `Net::HTTP` directly? If we're not already depending on `httparty`, we might want to do a brief analysis to ensure that adding this dependency is sound.

Reply:
That's a good point. I followed Feedjira docs on that one. It is easy enough to do away with `HTTParty.get`. It is easily replaced with `Net::HTTP::Get` request.